src/scenes/Activities/scenes/EventHistory.js

```diff
@@ -404,8 +406,8 @@ export default class EventHistory extends Component {
   }
 }
 
-const NameLinkCell = ({ classes, clickable, topic, ...props }) => (
+const NameLinkCell = ({ clickable, topic, ...props }) => (
   <TableCell>
-    <span className={clickable && classes.viewEventLink} {...props} />
```

Comment:
don't remove style, follow the design by adding corresponding styles. You can find the styles in previous commit.

Reply:
comment above